### PR TITLE
add new Container Scan workflow

### PR DESCRIPTION
**Pull request by mertel-rh**
I cleaned up and added the Container Scanning workflow that I worked on with @itewk 
Please make sure I didn't miss anything in my cleanup and that I added the vars correctly to the Map.

### CODE REVIEW

File: vars/ploigosWorkflowContainerScan.groovy [hunk 1]
```diff
@@ -0,0 +1,474 @@
...
+		                      usernameVariable: 'REGISTRY_USERNAME',
+                		      passwordVariable: 'REGISTRY_PASSWORD')]) {
+
+                                  sh """
+                                      if [ "${params.verbose}" == "true" ]; then set -x; else set +x; fi
+                                      set -eu -o pipefail
+                                      
+				      buildah pull --storage-driver=vfs --creds=${REGISTRY_USERNAME}:${REGISTRY_PASSWORD} ${params.registryURL}/${IMAGE_TARGET}
+				      buildah push --storage-driver=vfs ${params.registryURL}/${IMAGE_TARGET} docker-archive:/home/ploigos/${params.imageName}.tar
+                                     """
```
Comment (itewk) [inline vars/ploigosWorkflowContainerScan.groovy]: fix indent

File: vars/ploigosWorkflowContainerScan.groovy [hunk 2]
```diff
@@ -0,0 +1,474 @@
...
+    Variable for setting the Repository/Container of the container image to scan
+    i.e. quay.io/myorg/mycontainer:mytag imageName = "mycontainer"
+    */
+    string imageName = ''
+    
+    /*
+    Variable for setting the Repo/Org/Subdirectory of the container image to scan
+    i.e. quay.io/myorg/mycontainer:mytag imageTag = "mytag"
+    */
+    string imageTag = ''
```
Comment (itewk) [inline vars/ploigosWorkflowContainerScan.groovy]: capital S

File: vars/ploigosWorkflowContainerScan.groovy [hunk 3]
```diff
@@ -0,0 +1,474 @@
...
+    Variable for setting the Org/Subdirectory of the container image to scan
+    i.e. quay.io/myorg/mycontainer:mytag imageRepo = "myorg"
+    */
+    string imageOrg = ''
+    
+    /*
+    Variable for setting the Repository/Container of the container image to scan
+    i.e. quay.io/myorg/mycontainer:mytag imageName = "mycontainer"
+    */
+    string imageName = ''
```
Comment (itewk) [inline vars/ploigosWorkflowContainerScan.groovy]: capital S

File: vars/ploigosWorkflowContainerScan.groovy [hunk 4]
```diff
@@ -0,0 +1,474 @@
...
+    Variable for setting the name of the Jenkins Credentials that containe a
+    Username/Password pair for accessing your registry where the container image is hosted
+    */
+    string registryCredentialName = ''
+    
+    /*
+    Variable for setting the Org/Subdirectory of the container image to scan
+    i.e. quay.io/myorg/mycontainer:mytag imageRepo = "myorg"
+    */
+    string imageOrg = ''
```
Comment (itewk) [inline vars/ploigosWorkflowContainerScan.groovy]: capital S

File: vars/ploigosWorkflowContainerScan.groovy [hunk 5]
```diff
@@ -0,0 +1,474 @@
...
+    Variable for setting the Registry URL of the container image to scan
+    i.e. quay.io/myorg/mycontainer:mytag registryURL = "quay.io"
+    */
+    string registryURL = ''
+    
+    /*
+    Variable for setting the name of the Jenkins Credentials that containe a
+    Username/Password pair for accessing your registry where the container image is hosted
+    */
+    string registryCredentialName = ''
```
Comment (itewk) [inline vars/ploigosWorkflowContainerScan.groovy]: capital S

File: vars/ploigosWorkflowContainerScan.groovy [hunk 6]
```diff
@@ -0,0 +1,474 @@
...
+    Flag for setting toggling SSL Cert Verification in Git during the
+        Pip Install of Step Runner. Set to 'true' for skipping cert verification.
+    */
+    String gitTlsNoVerify = false
+    
+    /*
+    Variable for setting the Registry URL of the container image to scan
+    i.e. quay.io/myorg/mycontainer:mytag registryURL = "quay.io"
+    */
+    string registryURL = ''
```
Comment (itewk) [inline vars/ploigosWorkflowContainerScan.groovy]: capital S

File: vars/ploigosWorkflowContainerScan.groovy [hunk 7]
```diff
@@ -0,0 +1,474 @@
+#!/usr/bin/env groovy
```
Comment (itewk) [inline vars/ploigosWorkflowContainerScan.groovy]: lets do `ploigosWorkflowExistingContainerImageScan` to make it clear its for scanning an already built container image.

**Review by itewk — CHANGES_REQUESTED** [review]

**Review by itewk — APPROVED** [review]

**Review by BillBensing — APPROVED** [review]

**Pull request merged**